cobalt/site/docs/reference/starboard/modules/13/decode_target.md

```diff
@@ -11,17 +11,15 @@ An SbDecodeTarget can be passed into any function which decodes video or image
 data. This allows the application to allocate fast graphics memory, and have
 decoding done directly into this memory, avoiding unnecessary memory copies, and
 also avoiding pushing data between CPU and GPU memory unnecessarily.
-
-## SbDecodeTargetFormat
```

Comment:
These don't look right to me. The header markers are removed which will make the headers appear as regular text and also removes the extra newline which (due to markdown) make the text appear on the same row as the text above.

Do you know of any changes in the corresponding C++ file that would cause this?

Same for the similarly removed header markers below.

Reply:
Judging by the amount of changes, it seems like these haven't been generated in a while, so likely many c++ changes. The only one I made was in, which is what my aim is to update:
https://github.com/youtube/cobalt/pull/2905

then this PR is all generated from two commands:

docker run -it --mount type=bind,source=$(pwd),target=/code docsite
pre-commit run